src/Symfony/Component/Mailer/Bridge/Sendinblue/Transport/SendinblueSmtpTransport.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Sendinblue\Transport;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Mailer\Transport\Smtp\EsmtpTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * @author Yann LUCAS
+ */
+final class SendinblueSmtpTransport extends EsmtpTransport
+{
+    const SENDINBLUE_SMTP_HOST = 'smtp-relay.sendinblue.com';
```

Comment:
These 3 consts can be inlined IMHO or at least be marked private.

Reply:
Yes i changed for inline values. There is no other usages for those constants.